analytics/filesystem/file_module.go

```diff
@@ -93,6 +94,7 @@ func (f *FileLogger) LogNotificationEventObject(ne *analytics.NotificationEvent)
 // Shutdown the logger
 func (f *FileLogger) Shutdown() {
 	// clear all pending buffered data in case there is any
+	gglog.Info("[FileLogger] Shutdown, trying to flush buffer")
```

Comment:
Nitpick: can we reference this library as `glog` as it's done in other parts of the code?